### DISCUSSION TITLE
Linux Version

### DISCUSSION THREAD

**Discussion by imskanand** [SpotX-Official/SpotX#79] [Ideas]
Create same for Ubuntu or Linux

**Comment by amd64fox** [comment]
Check out this repository https://github.com/abba23/spotify-adblock

Here I do not plan to introduce support for linux yet, since it is difficult to support it when you do not use it 🤣

**Comment by amd64fox** [comment]
https://github.com/SpotX-CLI/SpotX-Win/discussions/188
